### PR TITLE
Make section READMEs consistent

### LINKED ISSUE #243: Improve sections' front page consistency
**What's the issue?**

Consistency currently varies for section pages, such as 4.5 and 4.8. Some section pages have a table of contents while others have a lengthy page giving a summary and other information on the general topic and test cases. In particular I noticed [4.11 Business Logic Testing], which gives a summary of each test (violating DRY principle) and even lists Tools. I'd like to see these pages made more consistent.

**How do we solve it?**
Come to an agreement on the format for section pages, to be included in the project style guide.

Would you like to be assigned to this issue?
_Check the box if you will submit a PR to fix this issue. Please read CONTRIBUTING.md._
- [x] Assign me, please!

### PR DESCRIPTION
**Pull request by victoriadrake**
- Close #243
- Make all README files TOC pages
- Move any relevant introduction text to an introduction page
- Clean up titles

This PR covers issue #243.

- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

### CODE REVIEW

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/README.md [hunk 1]
```diff
@@ -1,157 +1,21 @@
-# Testing for Business Logic
+# 4.1 Testing Business Logic
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/README.md]: 4.11 right?

Comment (victoriadrake) [reply document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/README.md]: Thanks!

**Comment by ThunderSon** [comment]
Instead of having `4.<id>_Introduction_<section>`, what about `4.<id>.0_Introduction_<section>`? Thinking if it might create some issues in rendering.

**Comment by victoriadrake** [comment]
> Instead of having `4.<id>_Introduction_<section>`, what about `4.<id>.0_Introduction_<section>`? Thinking if it might create some issues in rendering.

I assume you’re referring to the file names? In which case I’m ambivalent about it and would leave it up to whether it makes automation easier in any particular case.

**Comment by kingthorin** [comment]
I'm good with making them the 0th document per directory/section.

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Introduction_to_Business_Logic.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+# 4.11 Introduction to Business Logic
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Introduction_to_Business_Logic.md]: "Introduction to Business Logic Testing" ?

**Pull request merged**